src/Chapter2.hs

```diff
@@ -841,7 +858,8 @@ list.
 
 🕯 HINT: Use the 'cycle' function
 -}
-rotate = error "rotate: Not implemented!"
+rotate :: Int -> [a] -> [a]
+rotate k xs = take (length xs) . drop k $ cycle xs
```

Comment:
It is better to look into negative number `k` case separately, the current solution doesn't cover it 🙂

Reply:
Thank you for your pointing out! I fixed it!